newIDE/app/src/SceneEditor/index.js

```diff
@@ -1425,7 +1437,9 @@ export default class SceneEditor extends React.Component<Props, State> {
             onInstancesMoved={this._onInstancesMoved}
             onInstancesResized={this._onInstancesResized}
             onInstancesRotated={this._onInstancesRotated}
-            selectedObjectNames={this.state.selectedObjectNames}
+            selectedObjectNames={this.state.selectedObjects.map(e =>
```

Comment:
Same here, you can compute the array of object names at the start of the `render` method